### DISCUSSION TITLE
Purpose of SceneBuilderTest in production package?

### DISCUSSION THREAD

**Discussion by AlmasB** [gluonhq/scenebuilder#437] [Q&A]
It seems SceneBuilderTest is a manual system test as the IDE reports no usages of it in the production package of `app`. If so, would it make more sense to move this to the test package?

**Comment by Oliver-Loeffler** [comment]
As of now, I'd agree with moving it into the test source tree. There, it could be reworked to run as a JUnit test or we could even utilize TestFX to run it.

**Comment by Oliver-Loeffler** [comment]
As the class has been moved, we can close this thread. However, the question regarding the classes initial purpose remains open.
If there is indeed something like a test mode (as commented) this could be implemented in production source tree and triggered by a command line option - but would also be a topic to discuss..
